collect_app/src/main/java/org/odk/collect/android/adapters/FormListAdapter.java

```diff
@@ -103,12 +105,14 @@ public FormListAdapter(
         }
     }
 
-    private String getTimestampText(Date date) {
+    private String getTimestampText(Long dateOfCreation, Long dateOfLastAttachmentsUpdate) {
```

Comment:
Potentially a bad idea, but I was wondering if we should only make this change for the "Fill Blank Form" page and deprecate `FormListAdapter` (currently only used for "Delete Blank Forms" which won't be available in Match Exactly) with a goal of sharing `BlankFormListAdapter` or some of its render code? That way we're not implementing this change in two places (and need extra tests), one of which will probably disappear at some point. Good to get @lognaturel's input here.

Reply:
I don't know... it's already implemented and so I'm ok with keeping it but if it's something not important I can get rid of it too and hat's not a problem at all. Let's see what @lognaturel thinks.

Reply:
I would probably do it in both places as it's been done and come back to sharing once we have a planned change with the delete view. Don't feel very strongly but especially now that it's already done I think it makes sense to come back to it.